docs/geocoder/getFeatureReverse.mdx

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
This file is called `getFeatureReverse.mdx`, it should be named `getFeaturesReverse.mdx`, same as the function name.